opencga-app/app/scripts/azure/arm/azuredeploy.json

```diff
@@ -205,7 +205,7 @@
     "variables": {
         "avereEnabled": "[equals(parameters('nfsStorageOption'), 'AvereCluster')]",
         "azureFilesEnabled": "[equals(parameters('nfsStorageOption'), 'AzureFiles')]",
-        "webserverNamingPrefix": "[toLower(substring(concat('webservers', uniqueString(parameters('rgPrefix'))), 0, 16))]"
+        "webserverNamingPrefix": "webservers"

```

Comment:
I've be tempted to say we drop this variable entirely from the top level template and handle this in the base template. (Even thought I was the one who originally did this... I have no idea why as it doesn't match the other templates)